manifests/config.pp

```diff
@@ -33,12 +34,13 @@
   }
 
   if $manage_server_xml == 'augeas' {
-    $_tomcat_max_threads   = { maxThreads   => $tomcat_max_threads }
-    $_tomcat_accept_count  = { acceptCount  => $tomcat_accept_count }
-    $_tomcat_port          = { port         => $tomcat_port }
-    $_tomcat_redirect_port = { redirectPort => $tomcat_redirect_port }
+    $_tomcat_max_threads          = { maxThreads   => $tomcat_max_threads }
+    $_tomcat_max_http_header_size = { redirectPort => $tomcat_max_http_header_size }
```

Comment:
Isn't it supposed to be:

```suggestion
    $_tomcat_max_http_header_size = { maxHttpHeaderSize => $tomcat_max_http_header_size }
```

It looks like this augeas thing is not tested

Reply:
OMG, yes. Too much copy pasting... sorry I will fix it now